scenarios/kubernetes_kops_aws.py

```diff
@@ -165,6 +165,8 @@ def main(args):
                (args.cluster, zones, args.state, args.nodes, aws_ssh))
     if args.image:
         e2e_opt += ' --kops-image=%s' % args.image
+    if args.kops_args:
+        e2e_opt += ' --kops-args=\"%s\"' % args.kops_args
```

Comment:
Please use the flag directly (`cmd.append('--kops-args=K')`), do not rely on the magic E2E_OPT environment variable.